static_src/components/form.jsx

```diff
@@ -0,0 +1,257 @@
+
```

Comment:
If not, I think this element stuff should be spun of into a library so that others could use when starting their own React projects.